decompressed/gui_file/www/lang/it-it/webui-core.po

```diff
@@ -3175,7 +3175,7 @@ msgid "Remove Telstra GUI"
 msgstr "Rimuovi GUI Telstra"
 
 msgid "Change the default page in system-extra to set this gui as default"
-msgstr "Modifica la pagina predefinita in 'Sistema' -> 'Extra' per impostare questa GUI come predefinita"
+msgstr "Per impostare questa GUI come predefinita modifica la pagina predefinita in<br/>'Sistema' -> 'Extra'"
 
 msgid "Update blacklist app"
 msgstr "Aggiorna app 'Elenco non autorizzati'"
```

Comment:
so che non e' una modifica fatta in questo PR ma soprattutto quando ci riferiamo alla Blacklist come app installabile direi di non storpiargli il nome e lasciarla nominata come Blacklist

Reply:
Se il termine si riferisce ad una funzione è preferibile usare se possibile un termine italiano.
Se invece il termine si rifererisce ad un app che ha questo nome (own name) come mis embra in questo caso è corretto lasciare 'Blacklist'.
Secondo me visto che la stringa non riguarda questa PR direi se sei d'accordo di fare il merge di queta PR e farò un'altra PR con le modifiche su Blacklist.